### PR TITLE
#404 Tweet when a project is published

### PR DESCRIPTION
**Pull request by carlosmiranda**
#404: New projects being published will now be tweeted.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#454](https://github.com/zerocracy/farm/issues/454) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#454](https://github.com/zerocracy/farm/issues/454) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by fabriciofx** [comment]
@carlosmiranda Could you fix these conflicts? Thanks!

**Comment by carlosmiranda** [comment]
@fabriciofx merged, please check

**Comment by carlosmiranda** [comment]
@fabriciofx ping

**Comment by fabriciofx** [comment]
@yegor256 This PR seems fine. I accept it.

File: src/main/resources/com/zerocracy/stk/pm/publish_project.groovy [hunk 1]
```diff
@@ -51,6 +55,22 @@ def exec(Project project, XML xml) {
       .param('author', claim.author())
       .param('pid', project.pid())
       .postTo(new Pmo(farm))
+    String message = new Par('The project %s was published by @%s')
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pm/publish_project.groovy]: @carlosmiranda I don't think it's the right message for our Twitter account :) Let's say something like this:

```
A new project "%s" was published on our board just a minute ago, feel free to apply and join!
```

Here `%s` should be project title, not `pid()`. You can get it from `Catalog`.

**Comment by yegor256** [comment]
@carlosmiranda see above

File: src/main/resources/com/zerocracy/stk/pm/publish_project.groovy (hunk 1 above)
Comment (carlosmiranda) [reply src/main/resources/com/zerocracy/stk/pm/publish_project.groovy]: @yegor256 agreed, updated

**Comment by carlosmiranda** [comment]
@yegor256 pushed new commits, please check

**Comment by 0crat** [comment]
The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#454](https://github.com/zerocracy/farm/issues/454), please stop working

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#454` is not assigned, can't get start time

**Comment by 0crat** [comment]
The job [#454](https://github.com/zerocracy/farm/issues/454) is now out of scope

**Pull request merged**